infra/label_evaluator_step_functions/lambdas/unified_receipt_evaluator.py

```
@@ -1115,534 +1175,414 @@ async def run_geometric() -> tuple[dict, float]:
-                        except Exception as e:
-                            logger.warning(
-                                "Failed to close Chroma client cleanly: %s",
-                                e,
-                            )
                 review_duration = time.time() - review_start
 
+        # Close ChromaDB client after all phases
+        if chroma_client and hasattr(chroma_client, "close"):
+            try:
+                chroma_client.close()
+            except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)